Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreter.cpp

```diff
@@ -151,6 +151,14 @@ void CachedInterpreter::ExecuteOneBlock()
   }
 }
 
+std::size_t CachedInterpreter::GetHostCodeSize(const JitBlock& block)
+{
+  const Instruction* const begin = reinterpret_cast<const Instruction*>(block.normalEntry);
+  for (const Instruction* iter = begin;; ++iter)
+    if (iter->type == Instruction::Type::Abort)
+      return reinterpret_cast<const u8*>(iter + 1) - reinterpret_cast<const u8*>(begin);
```

Comment:
Faster and simpler way:

```suggestion
  return block.codeSize;
```

With this simplification, this probably doesn't even need to be a function.

Reply:
I am attempting to remove `JitBlockData::codeSize`, but cannot do so until my JitWidget refresh (that relies upon this PR) is merged in. As a data member, I find codeSize to be less clear than doing pointer arithmetic on the near/far begin/end pointers as needed.

Reply:
In that case, I would recommend making the cached interpreter set the near/far begin/end pointers so that you can use your usual approach for calculating the code size instead of having to iterate through the whole code like this. (The far pointers could just be nullptr/nullptr.)